client/my-sites/plans-features-main/index.jsx

```diff
@@ -379,6 +388,44 @@ export class PlansFeaturesMain extends Component {
 		return false;
 	}
 
+	renderProductsSelector() {
+		if ( ! isEnabled( 'plans/jetpack-backup' ) ) {
+			return null;
+		}
+
+		const { intervalType, siteId, translate } = this.props;
+		const products = [
+			{
+				title: 'Jetpack Backup',
+				description: (
+					<p>
+						Automatic scanning and one-click fixes keep your site one step ahead of security
+						threats. <a href="https://jetpack.com/">More info</a>
+					</p>
+				),
+				id: PRODUCT_JETPACK_BACKUP,
+				options: {
+					yearly: [ PRODUCT_JETPACK_BACKUP_DAILY, PRODUCT_JETPACK_BACKUP_REALTIME ],
+					monthly: [
+						PRODUCT_JETPACK_BACKUP_DAILY_MONTHLY,
+						PRODUCT_JETPACK_BACKUP_REALTIME_MONTHLY,
+					],
+				},
+				optionsLabel: 'Backup options',
+			},
+		];
+
+		return (
+			<Fragment>
+				<FormattedHeader
+					headerText={ translate( 'Single Products' ) }
+					subHeaderText={ translate( 'Just looking for a backups? We’ve got you covered.' ) }
+				/>
+				<ProductSelector products={ products } intervalType={ intervalType } siteId={ siteId } />
```

Comment:
Is the `siteId` needed? It was needed for the devdocs example, but here, ideally, we should be getting that from the currently selected site ID (type `state.ui.selectedSiteId` in your browser console while looking at the plans page to see if that's accessible).